README.md

```diff
@@ -1,8 +1,12 @@
 # ⚠️ Deprecated ⚠️
  
-`@most/sample` is deprecated. Only critical bug fixes will be released.
+`@most/sample` is deprecated.
 
-Its functionality is currently available in [@most/core](https://mostcore.readthedocs.io/en/latest/api.html#sample), and will also be available in most 2.0.
+Support and maintenance will cease when `@most/core` 1.0 is released. 
+
+Meanwhile, only critical bug fixes will be released. 
```

Comment:
Move up as a continuation of line 5, so there are no line breaks between the two sentences. It is strange to start a new paragraph with `Meanwhile, ...` because it is inherently connected with the sentence above.